src/server/db.js

```diff
@@ -0,0 +1,53 @@
+import mysql from 'mysql';
+import config from 'config';
+
+// URL of the MySQL server. We expect that the DB is listening on the
```

Comment:
Comments no longer valid, since everything was moved to config.

Reply:
No comment.
(see what I did there?)